core/app/c/[communitySlug]/stages/manage/components/panel/StagePanelSheet.tsx

```diff
@@ -28,6 +28,7 @@ export const StagePanelSheet = (props: Props) => {
 	return (
 		<Sheet open={open} onOpenChange={onOpenChange}>
 			<SheetContent className="max-h-100vh overflow-y-auto sm:max-w-md">
+				<SheetTitle className="sr-only">Edit Stage</SheetTitle>
```

Comment:
I added this as sr-only since it was to fix an accessibility warning, although I think it would be reasonable to have a visible title here too

Reply:
thanks! that error was driving me crazy